.gitlab-ci/stages/e2e.yml

```diff
@@ -9,11 +9,6 @@ Test on deployed branch:
     variables:
       # Don't run when deploying in production
       - $PRODUCTION
-  artifacts:
-    expire_in: 1 day
-    when: always
-    paths:
-      - optional/e2e/runners/puppeteer/output
```

Comment:
Is it really a problem ?
Those artifacts are the only visual hint we have today when a e2e test fails.

Reply:
Well, I have no idea how to retrieve that. Also A day artifact is too short, most of the time we have PR that live a couple of days

Reply:
> A day artifact is too short, most of the time we have PR that live a couple of days

Change it to 1 week maybe